contracts/README.md

```diff
@@ -22,7 +22,7 @@ A concrete implementation of `TeleporterTokenSource` and `IERC20Bridge` that han
 A concrete implementation of `TeleporterTokenSource` and `INativeTokenBridge` that handles the locking and release of the native EVM asset on the home chain. The optional Teleporter message fees used to incentivize a relayer for messages sent by this contract are all paid in a "wrapped native token" asset (e.g. WAVAX for the case in which AVAX is the native token). The contract address of the wrapped native token contract to be used must be provided in its constructor.
 
 ### `TeleporterTokenDestination`
-An abstract implementation of `ITeleporterTokenBridge` for a bridge contract on a destination chain that receives bridged assets from a `TeleporterTokenSource` instance. Handles receiving bridge messages from the specified token source contract to process token imports from the home chain (i.e mints), as well as burning tokens and sending messages to route them back to other chains. 
+An abstract implementation of `ITeleporterTokenBridge` for a bridge contract on a destination chain that receives bridged assets from a `TeleporterTokenSource` instance. Handles receiving bridge messages from the specified token source contract to process token imports from the home chain (i.e mints), as well as burning tokens and sending messages to route them back to other chains. Once deployed, `TeleporterTokenDestination` instances must be registered with their specified `TeleporterTokenSource` contracts. This is by calling `registerWithSource` on the destination contract, which will send a Teleporter message to the source contract with the message to register.
```

Comment:
```suggestion
An abstract implementation of `ITeleporterTokenBridge` for a bridge contract on a destination chain that receives bridged assets from a `TeleporterTokenSource` instance. Handles receiving bridge messages from the specified token source contract to process token imports from the home chain (i.e mints), as well as burning tokens and sending messages to route them back to other chains. Once deployed, a `TeleporterTokenDestination` instance must be registered with a specific `TeleporterTokenSource` contract. This is dont by calling `registerWithSource` on the destination contract, which will send a Teleporter message to the source contract with the message to register.
```

The wording as-is implied that a `TeleporterTokenDestination` may be registered with multiple sources.